client/common/cmdline.c

```diff
@@ -779,7 +777,7 @@ static char** freerdp_command_line_parse_comma_separated_values_offset(
 
 	p = t;
 
-	if (count)
+	if (*count)
```

Comment:
Checking pointer here is redundant because we already did it in `freerdp_command_line_parse_comma_separated_values ` with `assert(NULL != count);`